lib/utils/functions.ts

```diff
@@ -79,3 +79,39 @@ export function fold(text: string): string {
 
   return `${wrapped}\r\n`
 }
+
+/**
+  * Get the MIME type if a valid file ext is passed in
+  *
+  * @link  https://www.iana.org/assignments/media-types/media-types.xhtml#image
+  * @param string format
+  * @return string
+  */
+function getMimeFromFormat(format?: string): string {
+  switch (format) {
+    case 'jpeg':
+    case 'jpg':
+      return 'image/jpeg'
+    case 'png':
+    case 'gif':
+      return `image/${format}`
+    case 'svg':
+      return 'image/svg+xml'
+    default:
+      return ''
+  }
+}
+
+/**
+  * Get the MIME type from a URL, if possible
+  *
+  * @link   https://tools.ietf.org/html/rfc6350#section-6.2.4
```

Comment:
```suggestion
  * @link   https://tools.ietf.org/html/rfc2426#section-3.1.4
```

This library follows the [RFC 2426](https://tools.ietf.org/html/rfc2426) specification, for now.